core/frontend/src/render/webgl/BranchState.ts

```diff
@@ -46,6 +46,10 @@ export interface BranchStateOptions {
   forceViewCoords?: boolean;
   readonly viewAttachmentId?: Id64String;
   groupNodeId?: number;
+  /** If true, the view's [[DisplayStyleSettings.clipStyle]] will be disabled for this branch.
+   * No [[ClipStyle.insideColor]], [[ClipStyle.outsideColor]], or [[ClipStyle.intersectionStyle]] will be applied.
```

Comment:
These links are invalid - e.g., should be `[ClipStyle.insideColor]($common)` not `[[ClipStyle]]`.
@ben-polinsky docs build wouldn't catch this?

Reply:
Links have been updated to the above form.

Reply:
@pmconne It should but apparently there is an issue. I'll add it to the backlog but not sure when I'll have time to take a look.